### DISCUSSION TITLE
Repository Milestone Request in v4 GraphQL  API

### DISCUSSION THREAD

**Discussion by danjford** [community/community#22299] [New to GitHub]
<p>In the [V3 API](GET /repos/:owner/:repo/milestones) it is possible to do the following request:</p>
<pre><code>GET /repos/:owner/:repo/milestones
</code></pre>
<p>On this endpoint, you can then perform filters on the milestones based on a provided parameters e.g. state = <strong>open</strong> , <strong>closed</strong>  or <strong>all</strong>.</p>
<p>What is the equivalent of this in the V4 GraphQL API i.e. retrieving only <strong>OPEN</strong> milestones on a repository?</p>
<p><strong>UPDATE</strong></p>
<p>After some further research, this isn’t possible yet but it will be possible after changes are made following <a href="https://platform.github.community/t/schema-request-to-add-states-argument-for-the-milestones-connection/3958" rel="nofollow noopener">this schema request</a> raised in the platform community in November 2017.</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/danjford">@danjford</a> Thanks for being here!</p>
<p>This particular question might actually be better served in our Platform Community:</p>
<p><a href="https://platform.github.community/" rel="nofollow noopener">https://platform.github.community/</a></p>
<p>That forum is full of people who are experts on the GraphQL API and this question might actually already be answered there. I’d highly recommend giving that a shot if you don’t see an answer come in via this forum soon. </p>

**Comment by danjford** [comment]
<p>Thanks, I have opened the issue there instead.</p>
<p><a href="https://platform.github.community/t/repository-filtered-milestone-request/4569" rel="nofollow noopener">https://platform.github.community/t/repository-filtered-milestone-request/4569</a></p>

**Comment by isharailanga** [comment]
<p><a href="https://github.community/t/retrieve-open-milestones-for-a-repo/13804" title="Retrieve-open-milestones-for-a-repo">https://github.community/t5/GitHub-API-Development-and/Retrieve-open-milestones-for-a-repo/m-p/19817/thread-id/1009/highlight/false#M1010</a></p>
<p>This worked for me!</p>
